library/globals.inc.php

```diff
@@ -357,6 +357,19 @@
       xl('Show Insurance Address Information in the Insurance Panel of Demographics.')
     
   ),
+      'tiny_logo_1' => array(
+          xl('Mini Logo 1'),
+          'bool',                           // data type
+          '0',                              // default = false
+          xl('Show or hide mini logo 1 .')
+      ),
+
+      'tiny_logo_2' => array(
+          xl('Mini Logo 2'),
+          'bool',                           // data type
+          '0',                              // default = false
+          xl('Show or hide mini logo 2 .')
```

Comment:
No reason making above strings different since such a simple setting (and avoids making the translators do twice the work)